R.swift/SwiftTypes/Property.swift

```diff
@@ -12,6 +12,26 @@ protocol Property: UsedTypesProvider, CustomStringConvertible {
   var name: String { get }
 }
 
+struct AnyProperty: Property {
```

Comment:
The type-erasure is a nice solution, I would just use a function here I think:

```
func anyProperty(property: Property) -> Property {
  return property
}
```

Bit simpler and easier to maintain, but not a big deal. Will change this myself when the PR is merged and apply the type-erasure also in other parts of the code.